python/lsst/ts/m2gui/main_window.py

```diff
@@ -507,6 +517,33 @@ async def _callback_disconnect(self) -> None:
         action_disconnect.setEnabled(True)
         action_exit.setEnabled(True)
 
+    def _create_dialog_disconnect(self) -> QMessageBoxAsync:
+        """Create the disconnect dialog.
+
+        Returns
+        -------
+        dialog : `QMessageBoxAsync`
```

Comment:
Similar to a comment above, you might want to remove the return name?

Reply:
Same comment above. Thanks!